docs/configuration/other.md

```diff
@@ -15,6 +15,7 @@ api:
     ...
   api_security: !Jwt
       jwt_token 
+  default_max_records_num: 
```

Comment:
```suggestion
  default_max_records_num: 50
```
Change this so people know it's an ingeter

Reply:
addressed